engine/src/engine/renderer/Mesh.h

```diff
@@ -58,6 +58,8 @@ namespace Engine {
       VertexCategoryNormal = BIT(2),
     };
     explicit ObjFile() {}
+    // Warning	C26495	Variable 'Engine::ObjFile::file' is uninitialized. Always initialize a member variable (type.6).
```

Comment:
you should fix this before merging, @philsupertramp

Reply:
and while youre at it:https://github.com/godesteem/bucket-engine/blob/33fc1c2efa80dd522247ed21153a2a523f140a9b/engine/src/platform/opengl/OpenGLVertexArray.cpp#L62